man/systemd-networkd-wait-online.service.xml

```diff
@@ -85,6 +85,30 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-4</option></term>
+        <term><option>--ipv4</option></term>
+
+        <listitem><para>Waiting for an IPv4 address of each network interface in addition to the range of operational
+        states. If the argument <option>--any</option> will be passed, then <command>systemd-networkd-wait-online</command>
```

Comment:
> `If the argument <option>--any</option> will be passed`

Hmm??

Reply:
I meant, that the options --any + --ipv4 cause waiting for one IPv4 address of any link.

Reply:
> > `If the argument <option>--any</option> will be passed`
> 
> Hmm??

I have meant, that --any + --ipv4/--ipv6 options cause waiting only for one IPv4/IPv6 address of any link.

Reply:
> > `If the argument <option>--any</option> will be passed`
> 
> Hmm??

I have meant, that `--any` + `--ipv4`/`--ipv6` options cause waiting only for one IPv4/IPv6 address of any link.

Reply:
So, how about the following:
`If this option is specified with <option>--any</option>, then <command>systemd-networkd-wait-online</command> exits with success when at least one interface becomes online and has an IPv4 address.`